packages/core/src/components/StructuredMetadataTable/StructuredMetadataTable.test.tsx

```diff
@@ -31,7 +31,13 @@ describe('<StructuredMetadataTable />', () => {
 
   describe('Item Mappings', () => {
     it('Iterates over and displays every field in the map', () => {
-      const metadata = { field1: 'one', field2: 'two', field3: 'three' };
+      const metadata: {
+        [prop: string]: string;
+      } = {
+        field1: 'one',
+        field2: 'two',
+        field3: 'three',
+      };
```

Comment:
You might be able to just use `const metadata = { ... } as const` here instead. Doesn't always work and I didn't look at the rest of the code, but could give it a shot if you haven't already :grin: